langkit/themes.py

```diff
@@ -1,3 +1,12 @@
+"""
+References for jailbreaks and refusals
+- https://github.com/agencyenterprise/PromptInject
+- https://cdn.openai.com/papers/gpt-4.pdf
+- https://arxiv.org/pdf/2302.12173v1.pdf
+- https://github.com/gigabuck/prompt-rebellion
+
+
```

Comment:
nice to have a references blurb, but its not about this file so much as the json loaded? This set is more "example" than official, might make more sense as a text file next to themes.json.txt, can move in a follow up.